### ISSUE TITLE
Cannot test PostgreSQL trigger with ssh tunnel

### ISSUE THREAD

**Issue by DanFaudemer** [n8n-io/n8n#8970]
### Bug Description

When testing a PostgreSQL trigger with an ssh tunnel credential I get the following error : 

```
Problem in node ‘Postgres Trigger‘
connect ECONNREFUSED 127.0.0.1:5432
```

### To Reproduce

1. Create a PostgreSQL trigger 
2. Add a PostgreSQL credential with a ssh tunnel
3. Test the trigger

### Expected behavior

I should be able to test my trigger

### Operating System

Kubernetes

### n8n Version

1.33.1

### Node.js Version

18.19.1

### Database

PostgreSQL

### Execution mode

main (default)

**Comment by Joffcom** [comment]
Hey @DanFaudemer,

That is a good spot, it looks like when we added support for SSH tunnels we may not have updated the Postgres Trigger. I have created `NODE-1267` as the internal ticket to fix this.

**Comment by janober** [comment]
Fix got released with [`n8n@1.50.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.50.0)
